YumemiTraining/YumemiTraining/View/WeatherView.swift

```diff
@@ -0,0 +1,11 @@
+//
+//  WeatherView.swift
+//  YumemiTraining
+//
+//  Created by Daichi Private on 2021/04/10.
```

Comment:
[NITS]
Full name が違ってそう。

```suggestion
//  Created by Daichi Hayashi on 2021/04/10.
```

Reply:
8955c3b で修正